src/DurableTask.SqlServer.AzureFunctions/SqlTargetScaler.cs

```diff
@@ -12,14 +12,13 @@ public class SqlTargetScaler : ITargetScaler
     {
         readonly SqlMetricsProvider sqlMetricsProvider;
 
-        public SqlTargetScaler(string taskHubName, SqlMetricsProvider sqlMetricsProvider)
+        public SqlTargetScaler(string functionId, SqlMetricsProvider sqlMetricsProvider)
         {
             this.sqlMetricsProvider = sqlMetricsProvider;
 
             // Scalers in Durable Functions are shared for all functions in the same task hub.
             // So instead of using a function ID, we use the task hub name as the basis for the descriptor ID.
-            string id = $"DurableTask-SqlServer:{taskHubName ?? "default"}";
```

Comment:
Seems like we don't need this, but only the function id to create the target scaler descriptor. How we created this at AzureStorage ([here](https://github.com/Azure/azure-functions-durable-extension/blob/dev/src/WebJobs.Extensions.DurableTask/Listener/DurableTaskTargetScaler.cs#L32) )